### ISSUE TITLE
Macos Monterey

### ISSUE THREAD

**Issue by jonathasribeiro** [Vorshim92/Dell-Inspiron-5491-2-in-1#1]
is it possible to create an EFI for Monterey, or do you know of someone who is doing so?
I already have this Big Sur EFI, and I wanted to install monterey on my Note

**Comment by Vorshim92** [comment]
> is it possible to create an EFI for Monterey, or do you know of someone who is doing so?
> 
> I already have this Big Sur EFI, and I wanted to install monterey on my Note

Hi!yes. It's quite simple. I will send here tonight. For Monterey and newer there is only the wifi and Bluetooth driver to change and  also opencore of course. Do you have intel wifi or bcm?

**Comment by jonathasribeiro** [comment]
My wifi is BCM

**Comment by jonathasribeiro** [comment]
I'll wait for this EFI to come out and thank you very much for your attention

**Comment by Vorshim92** [comment]
> I'll wait for this EFI to come out and thank you very much for your attention

[EFI 085 monterey ventura.zip](https://github.com/Vorshim92/Dell-Inspiron-5491-2-in-1/files/9867326/EFI.085.monterey.ventura.zip)
here you are. try this. it's good also for ventura i think

**Comment by Vorshim92** [comment]
Don't forget to change serial and other stuff in platform info. I suggest to use [OCAuxiliaryTools](https://github.com/ic005k/OCAuxiliaryTools/releases/tag/20220235) 
Do you know how to change it?

**Comment by jonathasribeiro** [comment]
> Não se esqueça de alterar o serial e outras coisas nas informações da plataforma. Sugiro usar [OCauxiliaryTools.](https://github.com/ic005k/OCAuxiliaryTools/releases/tag/20220235) Você sabe como alterar?

no, do you have any tutorial explaining how to do it?

**Comment by Vorshim92** [comment]
> > Não se esqueça de alterar o serial e outras coisas nas informações da plataforma. Sugiro usar [OCauxiliaryTools.](https://github.com/ic005k/OCAuxiliaryTools/releases/tag/20220235) Você sabe como alterar?
> 
> 
> 
> no, do you have any tutorial explaining how to do it?

Did you ever change it? With other program?

**Comment by Vorshim92** [comment]
Did you already have an efi? On that efi what platform info do you have?

**Comment by jonathasribeiro** [comment]
> > > Não se esqueça de alterar o serial e outras coisas nas informações da plataforma. Sugiro usar OCauxiliaryTools [. ](https://github.com/ic005k/OCAuxiliaryTools/releases/tag/20220235)Você sabe como alterar?
> > 
> > 
> > não, você tem algum tutorial explicando como fazer?
> 
> Alguma vez você o mudou? Com outro programa?



No, I never got around to changing it, I always got an efi ready for my notebook model which is the Dell Inspiron 5491 2-in-1

**Comment by jonathasribeiro** [comment]
> Did you already have an efi? On that efi what platform info do you have?


No, I never got around to changing it, I always got an efi ready for my notebook model which is the Dell Inspiron 5491 2-in-1

**Comment by Vorshim92** [comment]
and where did you get your last efi? I suggest to learn before do anything... 
you can just take an EFI and use it without learn nothing. Can i suggest you to start from here [DORTANIA GUIDE](https://dortania.github.io/OpenCore-Install-Guide/config-laptop.plist/coffee-lake-plus.html)

**Comment by jonathasribeiro** [comment]
> and where did you get your last efi? I suggest to learn before do anything... you can just take an EFI and use it without learn nothing. Can i suggest you to start from here [DORTANIA GUIDE](https://dortania.github.io/OpenCore-Install-Guide/config-laptop.plist/coffee-lake-plus.html)

I get my last EFI in this repo: 
https://github.com/Speeedo83/Dell-Inspiron-5491-2in1-Hakintosh

it is exactly the same specs as my notebook

**Comment by Vorshim92** [comment]
> > and where did you get your last efi? I suggest to learn before do anything... you can just take an EFI and use it without learn nothing. Can i suggest you to start from here [DORTANIA GUIDE](https://dortania.github.io/OpenCore-Install-Guide/config-laptop.plist/coffee-lake-plus.html)
> 
> I get my last EFI in this repo: https://github.com/Speeedo83/Dell-Inspiron-5491-2in1-Hakintosh
> 
> it is exactly the same specs as my notebook

ye. it's from my friend Mario . We have the same laptop. Anyway, did you follow every instructions on his repo main page? 
if you use his last efi you and you didn't change the serial and other platform info here you did wrong xD 
![image](https://user-images.githubusercontent.com/7747380/198533472-1e879ca9-1121-4381-805a-a5a4355a56dd.png)

like this. (this image is from the EFI by Speeedo83 repo.).
One of the most important thing to do when you take an EFI from internet is to change platform info. Just pressing GENERATE button on OCAT. or using  [OPENCORE CONFIGURATOR](https://mackie100projects.altervista.org/download-opencore-configurator/) ... anyway. if you want to send me your EFI i will do it for you but send me in private your efi, not here

**Comment by jonathasribeiro** [comment]
I can install, but when the notebook restarts, there is no option to select the HD, only the macOS installer
